Snippets/Core/Core_6/Routing/InstanceMapping.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+namespace Core6.Routing
+{
+    using NServiceBus;
+
+    class InstanceMapping
+    {
+        void InstanceMappingFileConfig(EndpointConfiguration endpointConfiguration)
+        {
+            #region InstanceMappingFile-Config
```

Comment:
This snippet is referred to using the text "To map message types to logical endpoints, use the following config" and yet it's showing the instance mapping file API. Shouldn't that be divided out, and only show the `routing.RouteToEndpoint(...)` calls?